django_prices/widgets.py

```diff
@@ -2,24 +2,42 @@
 from django.template.loader import render_to_string
 from prices import Money
 
-__all__ = ["MoneyInput"]
+__all__ = ["MoneyInput", "MoneyConstCurrencyInput"]
 
 
-class MoneyInput(forms.TextInput):
+class MoneyInput(forms.MultiWidget):
+    def __init__(self, choices, attrs=None):
+        widgets = [
+            forms.TextInput(attrs={"type": "number", "step": "any"}),
+            forms.Select(choices=choices),
+        ]
+        super(MoneyInput, self).__init__(widgets, attrs)
+
+    def decompress(self, value):
+        if value and isinstance(value, Money):
+            return [value.amount, value.currency]
+        if value and isinstance(value, (list, tuple)) and len(value) == 2:
+            return value
+        return [None, None]
+
+
+class MoneyConstCurrencyInput(forms.MultiWidget):
```

Comment:
How about "fixed currency" instead of "const currency"? As in `FixedCurrencyMoneyInput`?

Reply:
It's an excellent idea; I really like that name.